react-elements/react-elements/dist/index.html

```diff
@@ -0,0 +1,2 @@
+<div id="root"></div>
+<script src="main.js"></script>
```

Comment:
You need the full html skeleton here. This will work but the browser fills in all the missing elements. We want to avoid the browser having to guess your intentions. Not all browsers will do it in the same way.